.github/workflows/pg-migtests.yml

```diff
@@ -14,6 +14,7 @@ jobs:
         EXPERIMENTAL_FAST_EXPORT: [0, 1]
     env:
       EXPERIMENTAL_FAST_EXPORT: ${{ matrix.EXPERIMENTAL_FAST_EXPORT }}
+    if: (github.event_name != 'pull_request') || (github.event_name== 'pull_request' && matrix.version == '2.18.0.0-b65')
```

Comment:
Can we introduce a variable like `LATEST_LTS_VERSION` and use it while comparing `matrix.version`?

If the `||` is short-circuit operator, then there is no need of the `github.event_name=='pull_request'` condition.